config/kraft/README.md

```diff
@@ -26,7 +27,7 @@ xtzWWN4bTjitpL3kfd9s5g
 ~~~~
 
 ## Format Storage Directories
-The next step is to format your storage directories.  If you are running in single-node mode, you can do this with one command:
+The next step is to format your storage directories.  If you are running in single-node mode, you can do this with one command. The string `<uuid>` should be replaced with the value returned by `kafka-storage.sh random-uuid`:
 
 ~~~~
 $ ./bin/kafka-storage.sh format -t <uuid> -c ./config/kraft/server.properties
```

Comment:
How about we put the last sentence as the command line comment. That is:

~~~~
# The string `<uuid>` should be replaced with the value returned by `kafka-storage.sh random-uuid`

$ ./bin/kafka-storage.sh format -t <uuid> -c ./config/kraft/server.properties
Formatting /tmp/kraft-combined-logs
~~~~